jekyll/_cci2_ja/authorize-google-cloud-sdk.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: classic-docs
+title: Google Cloud SDK の承認
+description: Google Cloud SDK を承認する方法
+categories:
+  - deploying
+redirect_from: /ja/google-auth
+version:
+  - クラウド
+  - Server 3.x
+  - Server 2.x
+---
+
+ここでは、プライマリ コンテナで [Google Cloud SDK](https://cloud.google.com/sdk/) をインストールおよび承認する方法を説明します。
+
+* 目次
+{:toc}
+
+## 概要
+{: #overview }
+{:.no_toc}
+
+Google Cloud SDK は、Google Cloud Platform (GCP) サービスへのアクセスに使用できる強力なツールセットであり、Google Compute Engine や Google Kubernetes Engine などが含まれます。 CircleCI では、GCP プロダクトにアプリケーションをデプロイする場合、Google Cloud SDK が推奨されます。
+
+## 前提条件
+{: #prerequisites }
+{:.no_toc}
+
+- CircleCI プロジェクト
+- GCP プロジェクト
+
+### Google Cloud SDK のインストール
+{: #installing-the-google-cloud-sdk }
+
+プライマリ コンテナでオペレーティング システムとして Debian を受け入れ可能な場合は、Google の基本 Docker イメージの使用を検討してください。 このイメージは、Docker Hub で [`google/cloud-sdk`](https://hub.docker.com/r/google/cloud-sdk/) として提供されています。
+
+それ以外の場合は、基本イメージのオペレーティング システムに対応する [Google Cloud SDK インストール手順](https://cloud.google.com/sdk/)に従ってください。
+
+### サービス アカウントの作成と格納
+{: #creating-and-storing-a-service-account }
+
+Google Cloud SDK のツールを使用する前に、`gcloud` を承認する必要があります。 Google は、ユーザー アカウントとサービス アカウントの 2 種類の承認を提供しています。 ここでは CircleCI に Cloud SDK をインストールするので、サービス アカウントが適切です。
+
+1. [Google の手順](https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account)の 1 ～ 3 に従ってサービス アカウントを作成します。 必ず JSON 形式のキー ファイルをダウンロードしてください。
+
+2. CircleCI にキー ファイルを[プロジェクト環境変数]({{ site.baseurl }}/ja/env-vars/#setting-an-environment-variable-in-a-project)として追加します。 この例では、`GCLOUD_SERVICE_KEY` という変数名になっています。 同じ名前を使用する必要はありませんが、このドキュメントのサンプルでは常にこの名前を使用します。
+
+3. 便宜上、次の 2 つの環境変数を CircleCI プロジェクトに追加します。
+    - `GOOGLE_PROJECT_ID`: GCP プロジェクトの ID
+    - `GOOGLE_COMPUTE_ZONE`: デフォルトの [Compute Engine のゾーン](https://cloud.google.com/compute/docs/regions-zones/)
+
+#### きめ細かい権限の追加
+{: #adding-granular-permissions }
+
+GCR へのコンテナ イメージのプッシュに問題が発生している場合は、デフォルトの `service account` によって提供されるものよりもきめ細かい権限が必要とされている可能性があります。 権限の変更は、Cloud Storage の [IAM コンソール](https://console.cloud.google.com/iam-admin/iam/project)で許可することができます。
+
+Refer to the Cloud Storage [permission documentation](https://cloud.google.com/storage/docs/access-control/iam-permissions) to learn more about Identity and Access Management (IAM) permissions.
```

Comment:
This sentence should be in Japanese.

Perhaps something like this?

```
Identity and Access Management (IAM) 権限の詳細については、Cloud Storage [permission documentation](https://cloud.google.com/storage/docs/access-control/iam-permissions) を参照してください。
```

Reply:
Thank you! Applied.